src/_guides/language/effective-dart/style.md

```diff
@@ -323,6 +315,16 @@ defaultTimeout
 kDefaultTimeout
 {% endprettify %}
 
+### DON'T explicitly name libraries
+
+Appending a name to the `library` directive is technically possible,
+but is a legacy feature and discouraged. 
+
+Dart generates a unique tag for each library
+based on its path and filename.
+Naming libraries overrides this generated URI, 
+which can make it harder for tools to physically find
+the main library file in question. 
```

Comment:
**issues:** The second sentence has two issues:
* "physically" is unneeded and not true; there is no physicality in code
* The sentence can be cut into two at "which".

Reply:
Addressed